lake-manifest.json

```diff
@@ -13,7 +13,7 @@
   {"url": "https://github.com/leanprover-community/quote4",
    "type": "git",
    "subDir": null,
-   "rev": "64365c656d5e1bffa127d2a1795f471529ee0178",
+   "rev": "53156671405fbbd5402ed17a79bd129b961bd8d6",
```

Comment:
(this is https://github.com/leanprover-community/quote4/compare/64365c656d5e1bffa127d2a1795f471529ee0178...53156671405fbbd5402ed17a79bd129b961bd8d6)

Reply:
I am confused. I just ran `lake update Qq` on the master branch

Reply:
Further the master branch was fetching a version without the manifest file

Reply:
Oh I see. The new commit at line 16 is the merge commit

Reply:
My intent here was to leave this comment chain visible so that someone can easily click through to see what the bump was.